src/Store.js

```diff
@@ -654,6 +653,7 @@ class Store {
     this.baseUrl = options.baseUrl || ''
     this.defaultFetchOptions = options.defaultFetchOptions || {}
     this.headersOfInterest = options.headersOfInterest || []
+    this.retryOptions = options.retryOptions || { attempts: 1, delay: 0 } // do not retry by default
```

Comment:
wouldn't we want it to retry once by default?

Reply:
we could! I was thinking of testing libraries and other things that may benefit from not needing to be configured to not spend extra time on requests but I'm open to other defaults.